src/components/CurriculumComponents/CurriculumDownloadTab/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC, useEffect, useLayoutEffect, useRef, useState } from "react";
+import { OakThemeProvider, oakDefaultTheme } from "@oaknational/oak-components";
+
+import CurriculumDownloadView, {
+  CurriculumDownloadViewData,
+} from "../CurriculumDownloadView";
+import { School } from "../CurriculumDownloadView/helper";
+import SuccessMessage from "../SuccessMessage";
+
+import {
+  saveDownloadsDataToLocalStorage,
+  useDownloadsLocalStorage,
+} from "./helper";
+
+import Box from "@/components/SharedComponents/Box";
+import { useFetch } from "@/hooks/useFetch";
+import { wrapPreRelease } from "@/hooks/usePrereleaseFlag";
+
+function ScrollIntoViewWhenVisisble({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const ref = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    if (ref.current) {
+      ref.current.scrollIntoView();
+    }
+  }, [ref]);
+  return <div ref={ref}>{children}</div>;
+}
+
+const CurriculumDownloadTab: FC = () => {
+  const { isLoading, data: localStorageData } = useDownloadsLocalStorage();
+  const [isDone, setIsDone] = useState(false);
+
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [data, setData] = useState<CurriculumDownloadViewData>(() => ({
+    schoolId: undefined,
+    schoolName: undefined,
+    email: undefined,
+    downloadType: "word",
+    termsAndConditions: false,
+    schoolNotListed: false,
+    schools: [],
+  }));
+
+  useLayoutEffect(() => {
+    if (localStorageData) {
+      setData({
+        schoolId: localStorageData.schoolId,
+        schoolName: localStorageData.schoolName,
+        email: localStorageData.email,
+        downloadType: "word",
+        termsAndConditions: localStorageData.termsAndConditions,
+        schoolNotListed: localStorageData.schoolNotListed,
+        schools: [],
+      });
+    }
+  }, [localStorageData]);
+
+  const schoolPickerInputValue = data.schoolName;
+  const { data: schoolList } = useFetch<School[]>(
+    `https://school-picker.thenational.academy/${schoolPickerInputValue}`,
+    "school-picker/fetch-suggestions",
+  );
+
+  const onSubmit = async (data: CurriculumDownloadViewData) => {
+    setIsSubmitting(true);
+    console.log("onSubmit", { data });
+
+    saveDownloadsDataToLocalStorage({
+      schoolId: data.schoolId!,
+      schoolName: data.schoolName!,
+      email: data.email!,
+      termsAndConditions: data.termsAndConditions!,
+      schoolNotListed: data.schoolNotListed!,
+    });
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
Would it be worth instead making `saveDownloadsDataToLocalStorage` an `async` function? The 2000ms timeout seems like it might backfire at some point

Reply:
Yeah this was just so James can see it working, while we haven't got a network request. But you're right I'll end up leaving this in by accident I'll remove it now

Reply:
I've remove this now